recipes/vtk/all/conanfile.py

```diff
@@ -58,26 +55,45 @@ def config_options(self):
             del self.options.fPIC
 
     def source(self):
-        if self.version is None:
-            self.version = "9.0.1"
         tools.get(**self.conan_data["sources"][self.version],
                   strip_root=True, destination=self._source_subfolder)
+    
+    def _convert_smp_option(self, v):
+        if v == "sequential":
+            return "Sequential"
+        elif v == "openmp":
+            return "OpenMP"
+        elif v == "tbb":
+            return "TBB"
+        elif v == "stl":
+            return "STDThread"
+        else:
+            raise "Invalid SMP Option"
```

Comment:
I recall this being better for the cli to handle

```suggestion
            raise ConanException("Invalid SMP Option")
```